modules/machineset-yaml-azure.adoc

```diff
@@ -118,7 +118,7 @@ endif::infra[]
           subnet: <infrastructure_id>-<role>-subnet <1> <2>
           userDataSecret:
             name: worker-user-data <2>
-          vmSize: qeci-22538-vnet
+          vmSize: D4s_v3
```

Comment:
Mike McCune happened to notice this was weird, fixed with 4.10 default value supplied by Kenny. Thanks, all!